File: social-carpooling-frontend/social-carpooling/src/app/core/model/trip.ts
import {Driver} from './driver';

export class Trip {
  id: string;
  date: string;
  time: string;
  from: string;
  destination: string;
  availablePlaces: number
  rdv: string;
  driver: Driver;

  constructor(){}


  /*constructor(date: string, time: string, from: string, destination: string, availablePlaces: number, rdv: string, driver: Driver) {
    this.date = date;
    this.time = time;
    this.from = from;
    this.destination = destination;
    this.availablePlaces = availablePlaces;
    this.rdv = rdv;
    this.driver = driver;
  }*/


}

File: social-carpooling-backend/src/main/java/tn/social/carpooling/core/model/Trip.java
package tn.social.carpooling.core.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class Trip {

    private long id;
    private LocalDate date;
    private String time;
    private String from;
    private String destination;
    private short availablePlaces;
    private String rdv;
}
